queue_services/entity-emailer/src/entity_emailer/email_templates/RES-PAID.jinja2

```diff
@@ -0,0 +1,22 @@
+{% extends 'RES-common.jinja2' %}
+
+{% block page_title %}You have successfully filed your restoration with the BC Business Registry.{% endblock %}
+{% block title_message %}You have successfully filed your restoration with the BC Business Registry.{% endblock %}
+
+{% block attachments %}
+    <ul>
+        <li>Restoration Application</li>
+        <li>Receipt</li>
+    </ul>
+{% endblock %}
+
+{% block after_processing %}
+    <p>
+        Once your filing has been processed, the business will be
+        registered, and the company will receive the below outputs:
+    </p>
+    <ul>
+        <li>Notice of Articles</li>
+        <li>Restoration Certificate</li>
```

Comment:
Certificate of Restoration

Reply:
... in Google Sheets, it was written as "Restoration Certificate", but I like your wording better.

Reply:
@vysakh-menon-aot the wording has been changed.  Please let me know if I can go-ahead and merge this PR.